src/toolchain-conventions.adoc

```diff
@@ -203,6 +203,65 @@ RISC-V psABI has a contain sections to describe the code model:
 - https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#large-code-model[Large code model]: -mcmodel=large
 - https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#medium-position-independent-code-model[Medium position independent code model]:  -fpic, -fPIC, -fpie or -fPIE.
 
+== Specifying tuning processor and tuning features with -mtune
+
+Compilers accept the `-mtune` flag to specify the tuning processor model and
+features. They allow users to compile with the assumption of certain performance
+characteristics. Tuning processors and tuning features should not affect any of
+the _architectural_ properties or behaviors.
+
+The `-mtune` flag accepts the following syntax: `-mtune=<tune-cpu>[":"<tune-features>]?`,
+in which
+[source]
+----
+tune-cpu      ::= 'tuning CPU name in lower case'
+tune-features ::= directive ["," directive]*
+----
+A *directive* can and can only _amend_ or _subtract_ a certain tuning
+feature from the tuning CPU. For example, given a tuning CPU `sifive-x280` that
```

Comment:
"Tuning CPU" is a slightly strange phrase, how about "tuning model", maybe something like:

```suggestion
A *directive* can and can only _enable_ or _disable_ a specific tuning
feature within the tuning model for the named CPU. For example, given a tuning 
model for the `sifive-x280` CPU that
```